spec/controllers/catalog_controller_spec.rb

```diff
@@ -112,4 +112,18 @@
       expect(assigns(:hathi_url)).to be_nil
     end
   end
+
+  describe 'home page' do
+    it 'renders empty searches for the home page' do
+      expect(controller).to have_received(:render_empty_search)
```

Comment:
I am only testing that the `render_empty_search` method was called in the Controller but not the logic inside of it nor the code inside `empty_solr_response` and `empty_raw_response`. Let me know if you think we should test deeper.